README.md

```diff
@@ -85,9 +85,97 @@ cartridge create --name myapp
+Now `rpm`, `deb`, `tgz`, `rock` and `docker` types of distributables are supported.
+
+There is one important detail about `rmp`, `deb` and `tgz` packing: for this types of packages rocks modules and executables specific for the system where `cartridge pack` command is running will be delivered.
+If you use `docker` packing, the result image will contain rocks modules and executables specific for the base image (`centos:8`).
+
+#### TGZ
+
+`cartridge pack tgz ./myapp` will create .tgz archive contains application source code and rocks modules described in application rockspec.
+
+#### RPM and DEB
+
+`cartridge pack rpm|deb ./myapp` will create RPM or DEB package.
+
+In case of opensource Tarantool package has `tarantool` dependency (version >= `<major>.<minor>` and < `<major+1>`, where `<major>.<minor>` is version of Tarantool used for application packing).
+You should enable Tarantool repo to allow your package manager install this dependency correctly.
+
+After package installation:
+
+* application code and rocks modules described in application rockspec will be placed in `/usr/share/tarantool/<app_name>` directory (for Tarantool Enterprise this directory will contain also `tarantool` and `tarantoolctl` binaries);
+
+* unit files for running application as a `systemd` service will be delivered in `/etc/systemd/system`;
+
+This directories will be created:
+
+* `/etc/tarantool/conf.d/` - directory for instances configuration;
+* `/var/lib/tarantool/` - directory to store instances snapshots;
+* `/var/run/tarantool/` - directory to store PID-files and console sockets.
+
+Read the [doc](https://www.tarantool.io/en/doc/2.2/book/cartridge/cartridge_dev/#deploying-an-application) to learn more about Tarantool Cartridge application deployment.
+
+To start the `instance-1` instance of the `myapp` service:
+
+```bash
+systemctl start myapp@instance-1
+```
+
+This instance will look up its [configuration](https://www.tarantool.io/en/doc/2.2/book/cartridge/cartridge_dev/#configuring-instances) across all sections of the YAML file(s) stored in /etc/tarantool/conf.d/*.
+
+#### Docker
+
+`cartridge pack docker ./myapp` will build docker image and tag it as `myapp:<version>-<patch>`.
+
+For Tarantool Enterprise you should specify download token using `--download_token` parameter or `TARANTOOL_DOWNLOAD_TOKEN` environment variable.
+It's needed to download SDK on result image.
+
+If you want `docker build` command to be runned with custom arguments, you can specify them using `TARANTOOL_DOCKER_BUILD_ARGS` environment variable.
+For example, `TARANTOOL_DOCKER_BUILD_ARGS='--no-cache --quiet'`
+
+Application code will be placed in `/usr/share/tarantool/${app_name}` directory.
+Opensource Tarantool will be installed on image.
+
+Run directory is `/var/run/tarantool/${app_name}`, workdir is `/var/lib/tarantool/${app_name}`.
+
+To start the `instance-1` instance of the `myapp` application:
+
+```bash
+docker run -d --env-file env.instance-1 \
```

Comment:
don't use --env-file here. it doesn't help to illustrate the example, but makes it actually harder. You even have to explain it below. just pass -e options as is